src/Storage/Sqlite.php

```diff
@@ -121,10 +119,10 @@ public function batch(array $data): bool
         return true;
     }
 
-    public function cleanup(\DateTime $now = null) : bool
+    public function cleanup(DateTimeImmutable $now = null): bool
     {
         if ($now === null) {
```

Comment:
You can rewrite the entire condition as `$now = $now ?? new DateTimeImmutable` .... and then remove, because you can move it to the `execute()`.